README.md

```diff
@@ -262,3 +262,16 @@ create table notes (
   title varchar(150) not null,
   text text not null
 );
+
+create table inventory_tracker (
+  id serial primary key,
+  username varchar (50) references users not null,
+  box_title varchar (500) not null
+);
+
+create table inventory_box_items (
+  id serial primary key,
+  inventory_box_id integer references inventory_boxes ON DELETE CASCADE not null,
+  item varchar (500) not null,
```

Comment:
This should probably be called name instead of item. I think I need to rename some columns in the other tables actually. Also, should we include a description column?